### PR TITLE
New branch for CC release notes

### PR DESCRIPTION
**Pull request by Amruta-Ranade**

### CODE REVIEW

File: releases/cockroachcloud-02022020.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+- CockroachCloud now requires that the password for a SQL user is at least 12 characters in length.
+- CockroachCloud now allows you to download the cluster's CA certificate directly from the shell instead of restricting the download functionality to a web browser.
+
+### General changes
+
+- Added a **Sign up** link to the [CockroachCloud **Log In** page](https://cockroachlabs.cloud/).
+- While [creating a new cluster](../stable/cockroachcloud-create-your-cluster.html), you can now type in the number of nodes you want in the cluster instead of having to click the `+` sign repeatedly.
+- The [**Create cluster**](../stable/cockroachcloud-create-your-cluster.html) page now displays the estimated hourly cost instead of the monthly cost.
+- Removed the cluster creation banner displayed at the top of the [**Clusters page**](cockroachcloud-cluster-management.html#view-clusters-page).
```
Comment (jseldess) [inline releases/cockroachcloud-02022020.md]: This link is broken. Use:

../v19.2/cockroachcloud-cluster-management.html#view-clusters-page

**Comment by jseldess** [comment]
___
*[releases/cockroachcloud-02022020.md, line 42 at r2](https://reviewable.io/reviews/cockroachdb/docs/6718#-M1RQB90EJguCEnmBXhL:-M1RQB90EJguCEnmBXhM:b-jbzxgg) ([raw file](https://github.com/cockroachdb/docs/blob/885a841e3a1ce0ab78e76dbf8700665d6194ee86/releases/cockroachcloud-02022020.md#L42)):*
> ```Markdown
> - Added language-specific connection string examples to the [Connect to your cluster](../v19.2/cockroachcloud-connect-to-your-cluster.html#use-a-postgres-driver-or-orm) document.
> - Added a tutorial on [streaming an enterprise changefeed from CockroachCloud to Snowflake](../v19.2/stream-changefeed-to-snowflake-aws.html).
> - Added a tutorial on [developing and deploying a multi-region web application](../v19.2/multi-region-overview.html).
> ```

This needs to be the full link to the tutorial in the crdb docs.

**Comment by jseldess** [comment]
___
*[_includes/sidebar-data-v20.1.cockroachcloud.json, line 169 at r2](https://reviewable.io/reviews/cockroachdb/docs/6718#-M1RQM1rA5Ecm5q_ZQPM:-M1RQM1rA5Ecm5q_ZQPN:b9pu0ae) ([raw file](https://github.com/cockroachdb/docs/blob/885a841e3a1ce0ab78e76dbf8700665d6194ee86/_includes/sidebar-data-v20.1.cockroachcloud.json#L169)):*
> ```JSON
>         "title": "March 2020",
>         "urls": [
>           "/releases/cockroachcloud-02022020.html"
> ```

Don't we want this to reflect today's date? If so, `02` at the start should be `03`.  Please change this in the other sidenav file and rename the release notes file itself as well.

**Comment by Amruta-Ranade** [comment]
> _[_includes/sidebar-data-v20.1.cockroachcloud.json, line 169 at r2](https://reviewable.io/reviews/cockroachdb/docs/6718#-M1RQM1rA5Ecm5q_ZQPM:-M1RQM1rA5Ecm5q_ZQPN:b9pu0ae) ([raw file](https://github.com/cockroachdb/docs/blob/885a841e3a1ce0ab78e76dbf8700665d6194ee86/_includes/sidebar-data-v20.1.cockroachcloud.json#L169)):_
> 
> > ```json
> >         "title": "March 2020",
> >         "urls": [
> >           "/releases/cockroachcloud-02022020.html"
> > ```
> 
> Don't we want this to reflect today's date? If so, `02` at the start should be `03`. Please change this in the other sidenav file and rename the release notes file itself as well.

Oooh..good catch. Fixed.

**Comment by Amruta-Ranade** [comment]
> _[releases/cockroachcloud-02022020.md, line 42 at r2](https://reviewable.io/reviews/cockroachdb/docs/6718#-M1RQB90EJguCEnmBXhL:-M1RQB90EJguCEnmBXhM:b-jbzxgg) ([raw file](https://github.com/cockroachdb/docs/blob/885a841e3a1ce0ab78e76dbf8700665d6194ee86/releases/cockroachcloud-02022020.md#L42)):_
> 
> > ```gfm
> > - Added language-specific connection string examples to the [Connect to your cluster](../v19.2/cockroachcloud-connect-to-your-cluster.html#use-a-postgres-driver-or-orm) document.
> > - Added a tutorial on [streaming an enterprise changefeed from CockroachCloud to Snowflake](../v19.2/stream-changefeed-to-snowflake-aws.html).
> > - Added a tutorial on [developing and deploying a multi-region web application](../v19.2/multi-region-overview.html).
> > ```
> 
> This needs to be the full link to the tutorial in the crdb docs.

Done.

**Pull request merged**